src/one_dragon/base/operation/application_base.py

```
@@ -64,26 +64,38 @@ def handle_init(self) -> None:
             pool.clear()
-            notify_level = self.ctx.notify_config.get_app_notify_level(self.app_id)
-            pool.max_images = 10 if notify_level >= NotifyLevel.ALL else 1
+            detail_mode = self.ctx.notify_config.get_app_detail_mode(self.app_id)
+            pool.max_images = 10 if detail_mode == NotifyDetailMode.MERGE.value.value else 1
 
         self.ctx.dispatch_event(ApplicationEventId.APPLICATION_START.value, self.app_id)
 
-    def after_operation_done(self, result: OperationResult):
+    def execute(self) -> OperationResult:
+        """
+        执行应用，并确保异常路径也退出 screen scope。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)